docarray/document/mixins/plot.py

```diff
@@ -92,7 +96,65 @@ def display(self, from_: Optional[str] = None):
         else:
             self.summary()
 
-    def display_tensor(self):
+    def _is_3d(self) -> bool:
+        """
+        Tells if Document stores a 3D object saved as point cloud or vertices and face.
+        :return: bool.
+        """
+        if self.uri and self.uri.endswith(tuple(MESH_FILE_EXTENSIONS)):
+            return True
+        elif self.tensor is not None and self.tensor.shape[1] == 3:
```

Comment:
Not sure if this work what if I have smth like this

np.zeros(1, 3 ,224 ,224) for an image ? It is still valid image have 3 channels